recipes-core/images/nilrt-proprietary.inc

```diff
@@ -1,3 +1,10 @@
+# Disable running ldconfig at the end of the rootfs creation process.
```

Comment:
Do we need a todo tag or a WI for this or would it just be obvious if we don't enable it?

Reply:
I think I had created a work item to address this during the Dunfell rebase. Let me see if I can find it and link it to this rebase.